<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login_manager</class>
 <widget class="QWidget" name="Login_manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="sign_in">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>300</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Зайти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sign_up">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>430</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать учетную запись</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>301</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>логин</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwdEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>301</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>пароль</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Авторизация</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
